### ISSUE TITLE
 Is it possible to improve the style of the buttons?

### ISSUE THREAD

**Issue by xdvi** [stephanrauh/ngx-extended-pdf-viewer#2609]
It is possible to improve the styling of the buttons by adding internal padding. Also, it would be better to implement a smoother hover effect without the focus ring, instead of using a ring when focusing, and reduce slightly the padding-right

![{7F1CA8B8-0E9E-4AEE-90C6-D44741B71B2C}](https://github.com/user-attachments/assets/ff198d3f-89cd-41ff-a217-c38a20b39989)

**Comment by stephanrauh** [comment]
Yeah, sure. What about this: you fix the layout in your global styles.css until you're satisfied and send me the result. If it passes my checks, I'll add it to the CSS code of ngx-extended-pdf-viewer. 

Alternatively, you can fix it in https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/main/projects/ngx-extended-pdf-viewer/src/lib/theme/common/viewer-with-images.scss and send me a pull request.

**Comment by yslflow** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by xdvi** [comment]
Hi Stephan,

I'm sorry for the delay in responding. Unfortunately, I haven't been able to run the project successfully on my local machine. Could you please guide me on how I can make the necessary changes to the CSS while being able to visualize the results? Any help would be greatly appreciated!

**Comment by stephanrauh** [comment]
And now I'm responding late... The build instructions are here: 
https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/main/projects/ngx-extended-pdf-viewer/how-to-build.md

I hope they're up-to-date. If something's missing, don't hesitate to tell me!
